### PR TITLE
Adding ToDo's (SteelGreatsword, Falconpots)

### PR DESCRIPTION
**Pull request by Tenscore**

### CODE REVIEW

File: Segments/Oakvael.mapproj [hunk 1]
```diff
@@ -118213,11 +118213,11 @@
           <block><![CDATA[]]></block>
         </script>
       </entry>
-      <entry weight="1">
+      <entry weight="4">
         <script name="OnCreate" enabled="true">
           <block><![CDATA[]]></block>
           <block><![CDATA[
-	return randomBalm(true);
```
Comment (Chilly021) [inline Segments/Oakvael.mapproj:118220]: Wouldn't you want to replace stamina pots with this? 

ItemEntity OnCreate(MobileEntity from, Container container)
{
	return new StaminaPotion();
}

I think Lux would want this for randomization of bad bottles. (orc/dragon/troll urine)

image

Comment (Chilly021) [reply Segments/Oakvael.mapproj:118220]: image

Comment (Tenscore) [reply Segments/Oakvael.mapproj:118220]: iirc, some random stam pots dropped alongside falcon stam pots.

The double entry on randombalms seemed a strange redundancy, and at 2.7% chance - seemed ideal for replacement.

Comment (Tenscore) [reply Segments/Oakvael.mapproj:118220]: ohhh.. thats the urine!  nm..   will pull it back.

**Pull request closed without merging**